diff --git a/pkg/sentry/platform/platforms/BUILD.bazel b/pkg/sentry/platform/platforms/BUILD.bazel
--- a/pkg/sentry/platform/platforms/BUILD.bazel
+++ b/pkg/sentry/platform/platforms/BUILD.bazel
@@ -5,6 +5,7 @@
     srcs = [
         "platforms.go",
         "platforms_darwin.go",
+        "platforms_debug.go",
     ],
     importpath = "gvisor.dev/gvisor/pkg/sentry/platform/platforms",
     visibility = ["//visibility:public"],