diff --git a/pkg/sentry/kernel/syslog.go b/pkg/sentry/kernel/syslog.go
index 44cb78465..3b728aec3 100644
--- a/pkg/sentry/kernel/syslog.go
+++ b/pkg/sentry/kernel/syslog.go
@@ -86,6 +86,11 @@ func (s *syslog) Log() []byte {
 		"Asking an AI agent to fix the bugs...",
 		"Rewriting the kernel in Rust...",
 		"Politicking the oom killer...",
+		"Supervising the supervisor...",
+		"Adding more KubernetesWorker nodes...",
+		"Picking a new name for Metropolis...",
+		"Re-reading the TPM spec...",
+		"Re-implementing mkfs.xfs in Go...",
 	}
 
 	selectMessage := func() string {
@@ -101,7 +106,7 @@ func (s *syslog) Log() []byte {
 
 	const format = "<6>[%11.6f] %s\n"
 
-	s.msg = append(s.msg, []byte(fmt.Sprintf(format, 0.0, "Starting gVisor..."))...)
+	s.msg = append(s.msg, []byte(fmt.Sprintf(format, 0.0, "Starting gVisor on Metropolis..."))...)
 
 	time := 0.1
 	for i := 0; i < 10; i++ {
